src/__template__/ComponentTemplate.vue

```diff
@@ -98,6 +98,7 @@ const rootElementStyles = computed((): Record<string, string> => ({
   height: iconSize.value,
   lineHeight: '0',
   width: iconSize.value,
+  ...(props.decorative && { pointerEvents: 'none' }),
```

Comment:
can you revert this back to the syntax you had previously? Was much easier to read
```suggestion
  pointerEvents: props.decorative ? 'none' : undefined,
```

Reply:
Getting type errors when trying to use this syntax (same with `null`):

![Screenshot 2024-10-30 at 11 17 02 AM](https://github.com/user-attachments/assets/b3d19821-a1b9-4665-afcf-477def40a31b)

I added a comment on that line though